gallery/src/components/NxToast/NxToastFullPageVariationsExample.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { useState} from 'react';
+import { reject, propEq } from 'ramda';
+
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxButton,
+  NxButtonBar,
+  NxFontAwesomeIcon,
+  NxBackButton,
+  NxStatefulGlobalSidebar,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxToastContainer,
+  NxToast,
+  NxSuccessAlert,
+  NxWarningAlert,
+  NxErrorAlert,
+  NxInfoAlert,
+  NxTile,
+  NxP
+} from '@sonatype/react-shared-components';
+import {
+  faArrowLeft,
+  faArrowRight,
+  faLink,
+  faQuestionCircle,
+  faBell,
+  faCog,
+  faUserCircle
+} from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
+
+type ToastTypes = 'success' | 'error' | 'info' | 'warning';
+
+interface ToastModel {
+  id: number;
+  type: ToastTypes;
+  message: string;
+}
+
+export default function NxToastFullPageVariationsExample() {
+  const [toastIdInc, setToastIdInc] = useState<number>(0);
+  const [toasts, setToasts] = useState<ToastModel[]>([]);
+
+  const addToast = (type: ToastTypes, message: string) => {
+    const toastId = toastIdInc + 1;
+    setToastIdInc(toastId);
+    setToasts([
+      ...toasts,
+      { id: toastId, type, message }
```

Comment:
New toasts are supposed to appear at the top of the list, right?

Reply:
>New toasts are supposed to appear at the top of the list, right?  This has them appearing at the bottom.

Yep, I fixed it here: [f7964c2](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/f7964c28bb7607880b52bccadd2b869d2cbea32d)